src/Core/Migration/V6_6/Migration1739355802FixVatHandling.php

```diff
@@ -0,0 +1,71 @@
+<?php declare(strict_types=1);
+
+namespace Shopware\Core\Migration\V6_6;
+
+use Doctrine\DBAL\Connection;
+use Shopware\Core\Framework\Log\Package;
+use Shopware\Core\Framework\Migration\MigrationStep;
+
+/**
+ * @internal
+ *
+ * @codeCoverageIgnore
+ */
+#[Package('framework')]
+class Migration1739355802FixVatHandling extends MigrationStep
+{
+    /**
+     * @return array<string, string>
+     */
+    private const VAT_PATTERNS = [
+        'BE' => 'BE\d{10}',
+        'CY' => 'CY\d{8}[A-Z]',
```

Comment:
already fixed, `Migration1736154963FixCyprusVatIdPattern` - maybe check the others too